examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>sugg: style as "...an <code>xgboost</code> model" or "...an XGBoost model"</p><p> </p><p>sugg: s/that will predict/that predicts/</p><p> </p><p>I find starting a sentence with inline code or a number awkward.  My sugg is "For the <code>rating_binary</code> column, a value of <code>1</code> indicates..."  I'd style the <code>0</code> as inline code too because it is like directed user input.  (Conversely, it's not that all numbers are inline code. It's OK to say the dataset has 100K records.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>